src/Tomcat/TomcatModel.ts

```diff
@@ -33,31 +33,50 @@ export class TomcatModel {
     public async saveServerList(): Promise<void> {
         try {
             await fse.outputJson(this._serversJsonFile, this._serverList.map((s: TomcatServer) => {
-                return { _name: s.getName(), _installPath: s.getInstallPath(), _storagePath: s.getStoragePath() };
+                return { _name: s.getName(), _installPath: s.getInstallPath(), _storagePath: s.getStoragePath(), _runInPlace: s.isRunInPlace() };
```

Comment:
The "Delete server" behavior had to change based on if the server was in-place or copied to the workspace, so I had to add this new param to the TomcatModel